eng/Versions.props

```diff
@@ -50,29 +50,28 @@
     <MicrosoftCodeAnalysisAnalyzersPackageVersion>3.12.0-beta1.25209.6</MicrosoftCodeAnalysisAnalyzersPackageVersion>
```

Comment:
```suggestion
    <MicrosoftCodeAnalysisAnalyzersPackageVersion>3.12.0-beta1.25222.12</MicrosoftCodeAnalysisAnalyzersPackageVersion>
```

Reply:
I don't understand how my local restore didn't find this issue 🤦‍♂️

Reply:
I think you'll have to update the details.xml as well

Reply:
oh yeah. okay